### ISSUE TITLE
e2e should catch subscriptions change payment problem

### ISSUE THREAD

**Issue by shendy-a8c** [Automattic/woocommerce-payments#3786]
There was a problem in changing subscriptions payment method that was caught by manual testing. Issues are here:
- https://github.com/Automattic/woocommerce-payments/issues/3765
- https://github.com/Automattic/woocommerce-payments/issues/3771

This issue is to make sure e2e test can catch this problem in the future.

Acceptance criteria:
- On a commit where the change payment issue, that is https://github.com/Automattic/woocommerce-payments/pull/3772, is reverted, run e2e test. Expect e2e test to fail.
- On latest commit where the fix of the change payment issue is effective, run e2e test. Expect e2e test to be green.

**Comment by frosso** [comment]
Closing as obsolete. This issue asked for e2e coverage so a future regression of #3772 would fail a test, and that coverage now exists at `tests/e2e/specs/subscriptions/shopper/shopper-subscriptions-manage-payments.spec.ts`.

The spec exercises both flows from the original bug: 'should change a default payment method to a new one' checks 'Use a new payment method' (the exact path flagged in #3765) and asserts 'Payment method updated.' plus the new card label, and a second test covers the saved-card path with the same assertion. If the #3772 fix regressed, these would fail - which is exactly the acceptance criteria here.

The spec runs in CI (tagged `@critical`, gated by the standard `shouldRunSubscriptionsTests`), and has no skip/fixme. The original bug was UPE-specific, and since UPE is now the only checkout path, the live test inherently runs the formerly-broken code. The bug, fix, and required coverage are 4+ years old and the test guards them today, so no code change is needed. I think we can close this.
